templates/upload.html

```diff
@@ -21,23 +21,39 @@ <h4 style='float: left;' for="file"> <b>Syllabus:</b> </h4>
       <h4 style='float: right;'> PDFs only</b> </h4>
       <input style="width:100%;" id="file" type="file" name="file" accept="application/pdf" required> <br>
     </div>
+    <div class="form-group group">
+      <h4 style='float: left;' for="file"> <b>Semester:</b> </h4>
+      <select name="semester" id="semester" style="margin: 1%;" required>
+        <option value="fall">Fall</option>
+        <option value="spring">Spring</option>
+        <option value="summer">Summer</option>
+      </select>
+      <select name="year" id="year" required>
```

Comment:
You're going to want to either make this a free-form input (that only accepts numbers), or have some way to automatically update the values so you don't have to come in here every year to add another option.